mods/tuxemon/maps/taba_town.tmx

```diff
@@ -375,11 +375,255 @@
     <property name="cond1" value="is variable_set battle_lost_faint:true"/>
    </properties>
   </object>
-  <object id="124" name="Tuxemart Sign" type="event" x="528" y="192" width="16" height="16">
+  <object id="125" name="bring on the nurses" type="event" x="352" y="640" width="16" height="16">
    <properties>
-    <property name="act1" value="dialog Tuxemart - A place to buy and sell items"/>
-    <property name="cond1" value="is player_facing_tile"/>
-    <property name="cond2" value="is button_pressed K_RETURN"/>
+    <property name="act1" value="create_npc nurse1,37,39,nurse,stand"/>
```

Comment:
more sprite names; i'll stop mentioning these, but please get rid of all of them.